apps/microtvm/reference-vm/zephyr/base-box/test-config.json

```diff
@@ -1,12 +1,14 @@
 {
     "stm32f746xx": {
         "vid_hex": "0483",
-        "pid_hex": "374b",
-        "test_cmd": ["pytest", "tests/micro/zephyr/test_zephyr.py", "--microtvm-platforms=stm32f746xx"]
+        "pid_hex": "374b"
         },
     "nrf5340dk": {
         "vid_hex": "1366",
-        "pid_hex": "1055",
-        "test_cmd": ["pytest", "tests/micro/zephyr/test_zephyr.py", "--microtvm-platforms=nrf5340dk"]
+        "pid_hex": "1055"
+        },
+    "mps2_an521": {
+        "vid_hex": "",
```

Comment:
it is a little confusing that "" means "don't attach anything" but i think that's ok.